app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -368,7 +368,7 @@ function update_fqdn() {
     .text();
   var pathname = window.location.pathname;
   var name = fqdn(host_name, domain_name);
-  if (name.length > 0 && pathname === '/hosts/new') {
+  if (name.length > 0 && pathname === tfm.tools.foremanUrl('/hosts/new')) {
```

Comment:
I think this should be without the `/` since the `URL_PREFIX` is `/`. So this may result in `//hosts/new`. Same above.

Reply:
interesting, locally i'm getting `''` as the URL_PREFIX

Reply:
Hmm, you're right. I was looking at the tests.

Reply:
checked on a production server as well and it also has `''`

Reply:
Should we update jest to match?
https://github.com/theforeman/foreman/blob/19fd5afc10b4c99f32ce115c4042e6fa5e0c9c22/config/jest.config.js#L24

Reply:
updated

Reply:
+1